### ISSUE TITLE
Build fails with pyrate_limiter >=3

### ISSUE THREAD

**Issue by Artturin** [ClementJ18/moddb#36]
```
> ImportError: cannot import name 'RequestRate' from 'pyrate_limiter' (/nix/store/ihs3g4hh44yl0fqhxm2f8rzvyjfmcwgk-python3.11-pyrate-limiter-3.1.0/lib/python3.11/site-packages/pyrate_limiter/__init__.py)
```

**Comment by Artturin** [comment]
Just renaming doesn't work

```
python3.11-moddb>     if isawaitable(clock.now()) or isawaitable(bucket.leak(0)):
python3.11-moddb>                    ^^^^^^^^^
python3.11-moddb> AttributeError: 'Rate' object has no attribute 'now'
```

https://github.com/vutran1710/PyrateLimiter/releases/tag/v3.0.0

**Comment by ClementJ18** [comment]
Will be fixed once https://github.com/ClementJ18/moddb/pull/38 has been merged into master. will probably release v0.9.1 for it

**Comment by ClementJ18** [comment]
Version has been pinned

Fixed in v0.10.0
